float32 rollAngle
float32 pitchAngle
float32 yawAngle
int32 thrust
